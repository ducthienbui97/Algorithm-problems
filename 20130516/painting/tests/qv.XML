<Exam Name="qv" TestScore="1" TestTimeout="1" ResultComparer="WordIgnoreCase">
  <Inputs>
    <Text>qv.INP</Text>
  </Inputs>
  <Outputs>
    <Text>qv.OUT</Text>
  </Outputs>
  <Tests>
    <Test ID="test01" Score="0" Timeout="0" />
    <Test ID="test02" Score="0" Timeout="0" />
    <Test ID="test03" Score="0" Timeout="0" />
    <Test ID="test04" Score="0" Timeout="0" />
    <Test ID="test05" Score="0" Timeout="0" />
    <Test ID="test06" Score="0" Timeout="0" />
    <Test ID="test07" Score="0" Timeout="0" />
  </Tests>
</Exam>