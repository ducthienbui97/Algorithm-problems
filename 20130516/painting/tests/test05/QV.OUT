15
598
5 492
